docs/src/__examples__/Pagination/DEFAULT.tsx

```diff
@@ -16,6 +21,21 @@ export default {
     {
       component: "Pagination",
       knobs: [
+        {
+          name: "labelPrev",
+          type: "text",
+          defaultValue: "Prev",
+        },
+        {
+          name: "labelNext",
+          type: "text",
+          defaultValue: "Next",
+        },
+        {
+          name: "labelProgress",
+          type: "text",
+          defaultValue: "Page 2 of 7",
```

Comment:
This is actually not being used. I would just remove it, because we can't be flexible to allow the construction of the whole string with the `currentPage` and `pageCount` variables…

And we can't make it be just for the `"of"` word, because that is not how the prop actually works, so it would be misleading. That's why I think we should not have this knob here.